fabfile.py

```
@@ -12,7 +12,17 @@
 
 
 def ssh(service):
-    local('docker exec -it etoolsinfra_%s_1 /bin/bash' % service)
+    # https://stackoverflow.com/a/25293275/8207
+    class NoBashException(Exception):
+        pass
+
+    with settings(abort_exception=NoBashException):
```

yeah, it will. 

I would think that any exception not related to bash would just fail hard again on the next line and the proper exception would bubble up anyways, but I suppose that is a bit misleading. I'm not aware of any better way to catch that exception specifically within `fabric`.

I'm fine not merging this if you think not worth it. Just seemed nice to provide a fallback in case bash isn't available on all containers and also to prevent requiring existing environments from rebuilding to run `fab ssh`.